<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonCrearVertice">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar Vertice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCrearVertice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>VERTICES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ARISTAS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOrigenArista">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Origen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDestinoArista">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Destino</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCrearArista">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEliminarArista">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonEliminarVertice">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>270</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Vertice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEliminarVertice">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>270</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VERTICES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPeso">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>531</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
